ras_backstage/resources/sign_in_uaa/sign_in_uaa.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from flask import request, make_response, jsonify
+from flask_restplus import fields, Resource
+from structlog import wrap_logger
+
+from ras_backstage import app, sign_in_uaa_api
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+sign_in_details = sign_in_uaa_api.model('SignInDetails', {
+        'username': fields.String(required=True, description='username'),
+        'password': fields.String(required=True, description='password')
+})
+
+
+@sign_in_uaa_api.route('/', methods=['POST'])
+class SignInUaa(Resource):
+
+    @staticmethod
+    @sign_in_uaa_api.expect(sign_in_details, validate=True)
+    def post():
+        logger.info('Retrieving sign-in details')
+        # force=true means there post doesn't HAVE to have application/json
```

Comment:
typo in docstring - there should be the I think

Reply:
Fixed